src/swdmt/redistricting/RegionTest.java

```diff
@@ -86,4 +88,40 @@ public void squareRegionNumberOfVotersTest() {
                        (new Region(i * i)).numberOfVoters(), is(i * i));
         }
     }
+    @Test(timeout = MAX_TIMEOUT)
+    public void regionWithVotersInsideLocations() {
```

Comment:
Recommend renaming method to `regionWithAllVotersInLocationSetTest`

Note: The names of all other test methods in this class end with "Test".